server/logic/data_parser.js

```diff
@@ -55,10 +55,10 @@ function parse (data, delimiter) {
     MsLogger.warn(errorStr)
   }
   const doesNotHaveTeamNumber = teamNumbersArray.some(team => {
-    return team == '' || team == undefined
+    return team == '' || team == undefined || parseInt(team) != undefined
```

Comment:
Test should be isNaN(parseInt(team))
probably return isNaN(parseInt(team)) is sufficient since
isNaN(parseInt('')) returns true
isNaN(parseInt(undefined)) returns true